ConversationServiceWatson/src/data/config.json

```diff
@@ -2,18 +2,19 @@
 	"api_key": "$(CONV_API_KEY)",
 	"watson_config": 
 	{
-		"endpoint": "https://gateway.watsonplatform.net/conversation/api/v1/workspaces/%s/message?version=2016-09-20",
+		"endpoint": "https://gateway.watsonplatform.net/conversation/api/v1/workspaces/%s/message?version=2021-11-27",
```

Comment:
I didn't test this code, but I remember that version 2021-11-27 had an issue.
Could you double-check if this works?
I think `2018-05-21` is the latest version that works with the current client implementation.